server.ts

```diff
@@ -19,7 +22,8 @@ export class Server {
   async create(req: JWTRequest, res: Response) {
     const todo: Todo = req.body;
     todo.ID = uuidv4();
-    todo.OwnerID = req.auth.sub;
+    const user = await this.directory.getUserByUserID(req.auth.sub);
```

Comment:
This doesn't solve our problem because it assumes that the JWT's subject name (the PID) is the same as the user key.
The PID (`req.auth.sub`) is the key of an _identity_ object with an `identity:identifier` relation to the user.
[See the Go implementation](https://github.com/aserto-demo/todo-go-v2/pull/4/files#diff-9f0a6b9e87ad313fe1a989af55b746d121544dc18d74eb97630277fefe2b8011R65) for reference. If calls GetRelation to get an ObjectIdentifier for the user associated with the identity and the calls GetObject with that identifier to retrieve the user object.

Reply:
Updated